providers/universal-provider/test/index.spec.ts

```diff
@@ -0,0 +1,329 @@
+import { describe, it } from "vitest";
+// import Web3 from "web3";
```

Comment:
Are tests coming latter on, or why are these commented out?

Reply:
Tests are on the way 🚄